tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/BuilderUtil.scala

```diff
@@ -77,6 +77,34 @@ object BuilderUtil {
     } else ret
   }
 
+  /**
+   * Some (variadic) operators introduce bound variables (e.g. map, funDef). This method constructs the expressions
+   * associated with the operator, and additionally performs shadowing checks and bound-variable tagging.
+   */
+  def boundVarIntroductionVariadic(
+      rawMethod: (TlaEx, Seq[(TlaEx, TlaEx)]) => TlaEx
```

Comment:
```suggestion
      uncheckedMethod: (TlaEx, Seq[(TlaEx, TlaEx)]) => TlaEx
```

I am not sure why it's called "raw"

Reply:
Old nomenclature. Someone requested `raw -> unsafe` at some point but it's still "raw" in my head apparently. I'll change this.